config/default.json

```diff
@@ -7,5 +7,8 @@
     "RESOURCE_SCOPUS_SEARCH_URL": "http://api.elsevier.com/content/search/scopus",
     "RESOURCE_SCOPUS_SEARCH_AUTHOR_URL": "http://api.elsevier.com/content/search/author",
     "RESOURCE_SCOPUS_RETRIEVE_AUTHOR_URL": "http://api.elsevier.com/content/author/author_id/",
-    "RESOURCE_SCOPUS_RETRIEVE_ABSTRACT_URL": "http://api.elsevier.com/content/abstract/doi/"
-}
+    "RESOURCE_SCOPUS_RETRIEVE_ABSTRACT_URL": "http://api.elsevier.com/content/abstract/doi/",
+    "RESOURCE_PUBMED_BASE_URL": "http://eutils.ncbi.nlm.nih.gov/entrez/eutils/",
+    "RESOURCE_PUBMED_SEARCH_PMID": "http://eutils.ncbi.nlm.nih.gov/entrez/eutils/esearch.fcgi?db=pubmed&field=title&retmax=100&retmode=json&term=",

```

Comment:
These are params and should be in the params object in the controller; db=pubmed, field=title etc